<launch>
  <node pkg="HRI_planner" type="human_localization" name="human_tracker">
  	<param name="camera_info_file" value="$(find HRI_planner)/camera_calibration/logitech_c310.yml"/>
	<remap from="/inside_out_tracker/image" to="/usb_cam/image_raw"/>
  </node>
</launch>
